08-projects.Rmd

```diff
@@ -5126,21 +5159,23 @@ id | name | capacity | address
 2  | Cary Quadrangle | 1200 | 1016 W Stadium Ave, West Lafayette, IN 47906
 3  | Hillenbrand Hall | NULL | 1301 3rd Street, West Lafayette, IN 47906
 
-**Relevant topics:** [sql](#sql), [inner join](#sql-inner-join)
+**Hint:** You can find how to typeset tables in the [RMarkdown cheatsheet](https://rstudio.com/wp-content/uploads/2016/03/rmarkdown-cheatsheet-2.0.pdf).
+
+**Relevant topics:** [SQL](#sql), [INNER JOIN](#sql-inner-join)
```

Comment:
Hi @kevinamstutz , it seems that `#sql-inner-join` is under development. Could you double check the section label?

Reply:
Thanks! Yes, I need to add a "join" section + sub-sections. I'll make sure this anchor link works.